src/commands/google.ts

```diff
@@ -68,9 +69,14 @@ const googleModule: CommandType = {
       })
       .then((data: Results) => {
         let i: number = 0;
-        data.items.forEach((element: SearchResult) => {
-          links.push(`${++i}. [${element.title}](${element.link})`);
-        });
+
+        if (data.items.length == 0) {
+          links.push("No results.");
+        } else {
+          data.items.forEach((element: SearchResult) => {
```

Comment:
If `items` is an array, this code is more expressive using `links = items.map((element) => ...);` instead.